src/app/compare/compare.service.ts

```diff
@@ -3,11 +3,18 @@ import { Subject } from 'rxjs/Subject';
 
 export const CompareService = {
   dialogOpen: false,
+  compareType: '',
   compareItem$: new Subject<{
     item: DimItem;
     dupes: boolean;
   }>(),
+  compareMultiItem$: new Subject<{
+    items: DimItem[];
```

Comment:
Rather than making a whole new observable, just make the original compareItem$ take an array of items.